manageiq-v2v.gemspec

```diff
@@ -9,13 +9,13 @@ Gem::Specification.new do |s|
   s.version     = ManageIQ::V2V::VERSION
   s.authors     = ['ManageIQ Authors']
   s.homepage    = 'http://www.manageiq.org'
-  s.summary     = 'ManageIQ plugin for v2v trasformations'
-  s.description = 'ManageIQ plugin for v2v trasformations'
+  s.summary     = 'ManageIQ plugin for v2v transformations'
+  s.description = 'ManageIQ plugin for v2v transformations'
   s.license     = 'MIT'
 
   s.files = Dir['{app,config,db,lib}/**/*', 'Rakefile', 'README.md']
 
-  s.add_dependency 'rails', '~> 5.0.0', '>= 5.0.0.1'
+  s.add_dependency 'rails', '>= 5.0.0.1', '< 5.2'
```

Comment:
Why do you need this at all?  You don't need the dependency, because this plugin will run as part of ManageIQ which already provides this dep.